packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -341,7 +341,7 @@ export interface IGetAvailableSlots {
 }
 
 export async function getAvailableSlots({ input, ctx }: GetScheduleOptions): Promise<IGetAvailableSlots> {
-  const orgDetails = input?.orgSlug
+  const orgDetails = input?.orgSlug !== undefined
```

Comment:
with the previous check, if orgSlug was `null`, it would still go to `orgDomainConfig`. Now it goes to 
```
{
        currentOrgDomain: input.orgSlug,
        isValidOrgDomain: !!input.orgSlug && !RESERVED_SUBDOMAINS.includes(input.orgSlug),
 }
```
and set `currentOrgDomain` to `null`, is `validOrgDomain` to `false`, which is what we expect.